specifyweb/frontend/js_src/lib/components/Molecules/FilePicker.tsx

```diff
@@ -140,6 +140,7 @@ export const downloadFile = async (
 ): Promise<void> =>
   new Promise((resolve) => {
     const iframe = document.createElement('iframe');
+    iframe.classList.add('absolute', 'hidden');
```

Comment:
I was hesitant to add `'hidden'` as some browsers might not trigger file downloads from hidden iframes.
Just adding `absolute` might be sufficient to fix this issue.

Reply:
`'hidden'` only affects visibility, not the placement. Presence or absence of scroll bar should not be affected by it.

Doesn't the scroll bar disappear as soon as you add `'absolute'`? If not, you might need to add `top-0` or something like that

Reply:
the 2nd video in your comment doesn't load for me, but I trust you

Interesting that there is not a single mention of scroll bars on the wiki page for visibility:hidden - https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

Reply:
keep the hidden. as otherwise there is no point in these changes if they don't fix the scroll bar